pyproject.toml

```diff
@@ -1,110 +1,110 @@
 ]
-license = { text = "BSD-3-Clause" }
 requires-python = ">=3.9"
 classifiers = [
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: BSD License",
     "Operating System :: OS Independent",
     "Programming Language :: C",
     "Programming Language :: Cython",
-    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: Implementation :: CPython",
     "Topic :: Scientific/Engineering :: Astronomy",
     "Topic :: Scientific/Engineering :: Physics",
 ]
-keywords = [
-    "astronomy",
-    "astrophysics",
-    "cosmology",
-    "space",
-    "science",
-    "units",
-    "table",
-    "wcs",
-    "samp",
-    "coordinate",
-    "fits",
-    "modeling",
-    "models",
-    "fitting",
-    "ascii",
+dynamic = [
+    "version",
 ]
 dependencies = [
-    "numpy>=1.22",
-    "pyerfa>=2.0",
     "astropy-iers-data>=0.2023.8.28.0.30.15",
+    "numpy>=1.22",
+    "packaging>=19",
+    "pyerfa>=2",
     "PyYAML>=3.13",
-    "packaging>=19.0",
 ]
-
 [project.optional-dependencies]
-test = [
-    "pytest>=7.0",
-    "pytest-doctestplus>=0.12",
-    "pytest-astropy-header>=0.2.1",
-    "pytest-astropy>=0.10",
-    "pytest-xdist",
-]
-test_all = [
-    "astropy[test]",  # installs the [test] dependencies
-    "objgraph",
-    "ipython>=4.2",
-    "coverage[toml]",
-    "skyfield>=1.20",
-    "sgp4>=2.3",
-]
-recommended = [
-    "scipy>=1.5",
-    "matplotlib>=3.3,!=3.4.0,!=3.5.2",
-]
 all = [
-    "astropy[recommended]",  # installs the [recommended] dependencies
+    "asdf-astropy>=0.3",
+    "astropy[recommended]", # installs the [recommended] dependencies
+    "beautifulsoup4",
+    "bleach",
+    "bottleneck",
     "certifi",
     "dask[array]",
+    "fsspec[http]>=2022.8.2",
     "h5py",
-    "pyarrow>=5.0.0",
-    "beautifulsoup4",
     "html5lib",
-    "bleach",
-    "pandas",
-    "sortedcontainers",
-    "pytz",
+    "ipython>=4.2",
     "jplephem",
     "mpmath",
-    "asdf-astropy>=0.3",
-    "bottleneck",
-    "ipython>=4.2",
-    "pytest>=7.0",
-    "typing_extensions>=3.10.0.1",
-    "fsspec[http]>=2022.8.2",
-    "s3fs>=2022.8.2",
+    "pandas",
     "pre-commit",
+    "pyarrow>=5",
+    "pytest>=7",
+    "pytz",
+    "s3fs>=2022.8.2",
+    "sortedcontainers",
+    "typing_extensions>=3.10.0.1",
 ]
 docs = [
-    "astropy[recommended]",  # installs the [recommended] dependencies
+    "astropy[recommended]", # installs the [recommended] dependencies
+    "Jinja2>=3",
+    "pytest>=7",
     "sphinx",
     "sphinx-astropy[confv2]>=1.9.1",
-    "pytest>=7.0",
-    "sphinx-changelog>=1.2.0",
+    "sphinx-changelog>=1.2",
     "sphinx_design",
-    "Jinja2>=3.0",
-    "tomli; python_version < '3.11'",
+    'tomli; python_version < "3.11"',
+]
+recommended = [
```

Comment:
Order again had (some) meaning here. That said, in this case I can see the advantage of just being alphabetical.